chainguard-dev/edu
# Intro grype

Added introduction to the grype container scanner.

### What should this PR do?

References https://github.com/chainguard-dev/internal/issues/3849

### How should this PR be tested?

grype is relatively straightforward as far as CLI tools go, but I'd like to make sure some of the extended command examples work, such as the examples that use pip and npm. We might also want to double check the appearance of the tutorial in the sidebar, as it's the first in the scanner series.
